internal_use/docs/source/BeginnerSegmentation/BeginnerSegmentationPlugins.md

```diff
@@ -157,94 +203,114 @@ This classifer was made in ilastik 1.4.1b5 by training on 4 images (A14_site1, E
 1. Add some very large annotations (a long, squiggly line, for example) to one image, then switch images in order to prove to yourself large annotations can actually harm in some cases rather than help (if you want to save your classifier, go back and use the eraser tool to delete these!).
-1. If you think you've sufficiently improved the prediction, save this model and return to CellProfiler. Did it help in the cases where you thought it would?
+1. If you think you've sufficiently improved the prediction, save this model and return to CellProfiler.
+Did it help in the cases where you thought it would?
 
 ## **Exercise 3: Running Cellpose from a Docker container**
 
-RunCellpose is by far our most popular plugin, simply because a) Cellpose is awesome! and b) `conda` installing software when you aren't very computationally comfortable isn't. You can use the plugin in either of two modes - using a local `conda` or `python` installation that contains both CellProfiler and Cellpose, OR using Docker. The run time with Docker is substantially slower (about a minute more per image, in our testing), but if installation would take you a long time and be frustrating, in this sense you can "trade" your personal hands-on frustration time for time where CellProfiler is running on your computer (but you aren't there). For many people, this is a good trade!
+RunCellpose is by far our most popular plugin, simply because a) Cellpose is awesome! and b) `conda` installing software when you aren't very computationally comfortable isn't.
+You can use the plugin in either of two modes - using a local `conda` or `python` installation that contains both CellProfiler and Cellpose in the same environment, OR using Docker.
+The module is quite a bit slower running Cellpose in Docker but this is a worthwhile trade-off to the challenge and frustration of installation for many people!
 
 ### Start Docker Desktop
+
 1. If you have not already installed Docker Desktop from the link above, please do so! This may involve rebooting your computer.
 1. Start Docker Desktop.
-1. Optional but strongly recommended - once Docker Desktop opens, use the search functionality to search `Cellpose` and look for the `cellprofiler/runcellpose_with_pretrained` model and pull it. If for whatever reason this isn't working, move on, but it will save you some time later. 
+1. Optional but strongly recommended - once Docker Desktop opens, use the search functionality to search `Cellpose` and look for a `cellprofiler/runcellpose_with_pretrained` model and pull it (if you have not already done so).
+If for whatever reason this isn't working, move on, but it will save you some time later.
 
 ### Grab the RunCellpose plugin
+
 1. Download [the plugin](https://github.com/CellProfiler/CellProfiler-plugins/blob/11b46ee7f6eb78f97f784a731c5a1931b66c90d4/active_plugins/runcellpose.py) into a folder on your local computer. As stated above, we strongly suggest a folder that contains ONLY plugins.
```

Comment:
@bethac07 Do you know why instructions say to download from a specific (old) commit? Do we need to check that the RunCellpose module in the repo pipeline upgrades properly when opened with current main?

Reply:
Should be fine to put it to master now, I think there was a concern back when I wrote it with the at-the-time newest version and I forgot to put it back to master later. 

Should open fine, but never hurts to check :)